_posts/brianna-stedman/09-09-2020-brianna-stedman-logical-turtle.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: brianna-stedman
+title: "Brianna's logical turtle"
+---
+
+What is Tina's favorite color? Give it your best guess! If you're correct, she'll do a little happy dance. 
+
+<iframe src="https://trinket.io/embed/python/6ea79a3db4" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
I did the same thing, I hope this is what he wanted!

Reply:
okay sweet, I think it's good to go if you wanna approve the change and merge it!